src/main/webapp/WEB-INF/static/js/trialmanager/germplasm-study-source/germplasmStudySource.js

```diff
@@ -70,6 +64,13 @@
 					}])
 					.withPaginationType('full_numbers');
 
+				function getPageQueryParameters(data) {
```

Comment:
Any way to make `getPageQueryParameters` a common function among the js files?

Reply:
I actually attempted to make this a common method, but chose not too because of the difference in implementation. for example in Inventory Tab, the order by value is based on the column name but in Sub Observation Tab, the order by value is the termid.